float32 aout0
float32 aout1
